### DISCUSSION TITLE
[Feature Request] RTL support in PrimeVue

### DISCUSSION THREAD

**Discussion by Abdullah0f** [primefaces/community#277] [PrimeVue]
Hello everyone, i really like PrimeVue and i find it one of the best components libraries in vue, but it still lack of RTL support which make users who are interested with RTL to go with another library just because the support of RTL

I really hope that you support it ASAP

**Comment by tugcekucukoglu** [comment]
Definitely, on our roadmap!

**Reply by Imadozone** [reply]
Guys, when should we have RTL support? I have seen the roadmap on PrimeVue site; RTL support had been scheduled for 2023 Q4. 
But we, as lovers of this component library, do not see any progress ☹
